Bio/Alphabet/__init__.py

```diff
@@ -19,6 +19,8 @@
 
 import warnings
 
+from typing import List, Optional, Union
```

Comment:
To avoid merge conflicts, please remove the changes to `` Bio/Alphabet/`` - it is no longer used, and likely to be removed or at best replaced with a stub very soon.

Reply:
Done. Thanks